experiments.json

```diff
@@ -238,5 +238,20 @@
       "threshold": 0.2,
       "description": "Edit Top Sites is enabled"
     }
+  },
+  "pocketStories": {
+    "name": "Trending stories provided by Pocket",
+    "active": true,
+    "description": "Display recommendations of trending stories by Pocket",
+    "control": {
+      "value": true,
+      "description": "Pocket trending stories active"
+    },
+    "variant": {
+      "id": "exp-017-pocketstories",
+      "value": true,
+      "threshold": 0,
```

Comment:
Please change this threshold to a non zero value, we usually use 0.2 as a reasonable size given our TestPilot population size.  If the experiment is not ready to be deployed, then set the threshold to 0.2 and set active to false.

Thanks!